public/includes/da-newsletter.php

```diff
@@ -0,0 +1,134 @@
+<?php
+class DrawAttention_Newsletter {
+	public $parent;
+	public $plugin_directory;
+
+
+	public function __construct( $parent ) {
+		$this->plugin_directory = DrawAttention::get_plugin_url() . '/public/';
+		$this->parent           = $parent;
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_meta_box_assets' ) );
+		add_action( 'admin_footer', array( $this, 'newsletter_modal_dialog' ) );
+		add_action( 'add_meta_boxes', array( $this, 'add_newsletter_widget' ) );
+
+		// Order the meta boxes
+		add_action( 'do_meta_boxes', array( $this, 'set_meta_boxes_position' ) );
+	}
+
+	public function enqueue_meta_box_assets() {
+		wp_enqueue_style( 'custom-meta-box-styles', $this->plugin_directory . '/assets/css/custom-meta-box-styles.css', array(), DrawAttention::VERSION );
```

Comment:
@Majed-Habli we need to namespace scripts/styles since this will cause conflicts if any other plugin has a style `custom-meta-box-styles`, so this will need to become `da-custom-meta-box-styles` wherever it's registered and enqueued